buildpipeline/tests/test_pipelines.json

```diff
@@ -190,6 +190,41 @@
                 "Type": "build/product/",
                 "PB_BuildType": "Release"
                 }
+            },
+            {
+                "Name": "Dotnet-CoreClr-Trusted-BuildTests",
+                "Parameters": {
+                "HelixJobType": "test/functional/cli/",
+                "TargetsWindows": "false",
+                "Rid": "alpine.3.6",
+                "TargetQueues": "Alpine.36.Amd64",
+                "TestContainerSuffix": "alpine36",
+                "TargetsNonWindowsArg": "TargetsNonWindows "
```

Comment:
Is this space intentional?  (several instances)

Reply:
I think it's because there is no space between TargetsNonWindowsArg and CrossGenArg here:

https://github.com/dotnet/coreclr/blob/0dc6bc8f13827d0bb432df775bf4bf292eaeac61/buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json#L128

If possible, there should be a space there, same with CrossGenArg